### DISCUSSION TITLE
Quick Responses Endpoints

### DISCUSSION THREAD

**Discussion by kmtchl** [dgreif/ring#1416] [Q&A]
Hello 🙂

Is there anyway to modify the selected Quick Responses with the Ring Doorbell via the API?

Some examples would be either turning it off/on. Changing the selected quick response etc.

Thanks!

**Comment by tsightler** [comment]
Not currently and not planned.

**Reply by kmtchl** [reply]
No problem. Am I correct in thinking Ring don't even expose it as an option? 

Thanks.

**Reply by tsightler** [reply]
If you can do it in the Ring app, then it's exposed, but of course someone has to research the required API calls.  I've looked at it a little, I believe it would be possible for sure.
